tools/debian_buildenv.sh

```diff
@@ -80,6 +80,7 @@ case "$1" in
             libusb-1.0-0-dev \
             libwavpack-dev \
             markdown \
+            ninja-build \
```

Comment:
I don't think we should use Ninja on Linux. Let's stick with the default main. 
At least this is not related to this PR. We may discuss it in a separate PR.

Reply:
Before this was handled by the buildenv scripts. But now those are not used. It is simpler to just add `-G Ninja` for all conditions.

Reply:
We can make it conditionally. But again, this change is unrelated to this PR.